dbms/src/Debug/dbgFuncMockRaftSnapshot.cpp

```diff
@@ -245,7 +245,8 @@ void MockRaftCommand::dbgFuncRegionSnapshot(Context & context, const ASTs & args
     auto end_decoded_key = RecordKVFormat::decodeTiKVKey(end_key);
 
     // Mock to apply an empty snapshot for region[region-id]
-    tmt.getKVStore()->handleApplySnapshot(
+    RegionBench::handleApplySnapshot(
```

Comment:
why move it to `RegionBench`?

Reply:
This is just a debug function and what this function call (`preHandleSnapshotToFiles`/`applyPreHandledSnapshot`) are public methods. So I want to move this debugging function out of KVStore to reduce the code complexity on KVStore